types/parse/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for parse 2.1.0
+// Type definitions for parse 2.1.0 and parse-server 3.1.3
```

Comment:
```suggestion
// Type definitions for parse 2.1.0
```
The type definitions comment line should only include the package and version of the actual package. If `parse-server` has a dependency on `parse`, that should be reflected in the type definitions for `parse-server`.

Reply:
@AlexandreHetu let's just remove the comment, looks like it would be most effective approach